### ISSUE TITLE
Syn_seed does not work in Sampling implementations 

### ISSUE THREAD

**Issue by wangxiaoyunNV** [dmlc/dgl#1844]
## 🐛 Bug

In examples/mxnet/sampling folder, graphsage_cv model, the default syn_seed is always 42, but each time it gives a different result. I also check where "syn_seed" is implemented, but I did not find how it is implemented. I also try to add random seed by myself, it did not change the syn_seed value, and it did not work. 

## To Reproduce
DGLBACKEND=mxnet python3 train.py --model graphsage_cv --batch-size 1000 --test-batch-size 5000 --n-epochs 50 --dataset reddit --num-neighbors 1 --n-hidden 128 --dropout 0.2 --weight-decay 0 --gpu 0

1.run the command above for several times. 
1.
1.

## Expected behavior

Since the syn_seed is always 42, I expect it will give the same acc everytime. 
## Environment

 - DGL Version (e.g., 1.0):'0.4'
 - Backend Library & Version (e.g., PyTorch 0.4.1, MXNet/Gluon 1.3): mxnet 1.6.0
 - OS (e.g., Linux):linux
 - How you installed DGL (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:3.6.9
 - CUDA/cuDNN version (if applicable):cuda11
 - GPU models and configuration (e.g. V100):RTX 8000
 - Any other relevant information: I use nvidia mxnet docker
